content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -218,6 +224,85 @@ Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number
 dogstatsd_so_rcvbuf: 26214400
 ```
 
+### Over UDS (Unix Domain Socket)
+
+#### Linux
+
+For UDS sockets, Linux is internally buffering datagrams in a queue if the
+reader is slower than the writer. The size of this queue represents the maximum number
+of datagrams that Linux will buffer per socket. This value can be queried with the following command:
+
+```bash
+sysctl net.unix.max_dgram_qlen
+```
+
+If the value is < 512, you can increase it to 512 or more using this command:
+
+```bash
+sysctl -w net.unix.max_dgram_qlen=512
+```
+
+Add the following configuration to `/etc/sysctl.conf` to make this change permanent:
+
+```conf
+net.unix.max_dgram_qlen = 512
+```
+
+In the same manner, the `net.core.wmem_max` could be incremented to 4MiB to improve
+client writing performances:
+
+```conf
+net.core.wmem_max = 4194304
+```
+
+Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number in `datadog.yaml`:
+
+```yaml
+dogstatsd_so_rcvbuf: 4194304
+```
+
+Please note that setting these values will need a reboot of the host.
+
+### Ensure proper packet sizes
+
+Sending packets with an adequate size to Dogstatsd will avoid extra work to the server.
+
+If the packets sent are too small, Dogstatsd will have to pack several together in order
+to process them in batches later in the pipeline. The official Dogstatsd clients are
+capable of grouping metrics to have the best ratio of the number of metrics per packet.
+
+If the clients properly send packets having a size close to `dogstatsd_buffer_size`, Dogstatsd
+will perform better. Be careful: the packets must not be larger than this size, otherwise, the
+Dogstatsd server won't be able to load them completely in the buffer and some of metrics will
+be malformed. Please use the corresponding configuration field in your Dogstatsd clients.
+
+Note for UDP: as the UDP packets are most of the time going through the Ethernet and IP layer,
+it is be better to avoid IP packets fragmentation by limiting the packet size to a value lower
+than a single Ethernet frame on your network. Most of the time, IPv4 networks are configured with
+a MTU of 1500 bytes, in this situation the packet size of sent packets should be limited to 1472.
+
+Note for UDS: for the best performances, UDS packet should have a size of 8192 bytes.
+
+### Limit the maximum memory usage
+
+Dogstatsd tries to absorb burst of metrics sent by the clients but in order to do so,
```

Comment:
AFAIK the terminology we use in our public docs is:

* `Dogstatsd`: the protocol
* the `Agent` (or `Dogstatsd-standalone binary`): the dogstatsd server
* `Dogstatsd clients`: (obviously) the clients :)

So in this section, I think we should replace most (all?) instances of `Dogstatsd` with `Agent`. If we're talking about the memory usage of the Dogstatsd bits of the Agent, I guess we can write `the Dogstatsd server within the Agent`, or something of that kind, WDYT?

Reply:
👍